### PR TITLE
Peace Hypospray, Escalation Module

### PR DESCRIPTION
**Pull request by DangerRevolution**
## About the PR

Adds the Peace Hypospray from SS13 Peacekeepers, adds an anti-fauna option based on Feedback from Players

## Why / Balance

Peace Hypospray = Non-lethal solution
Truncheon = anti-fauna

## Media

**Changelog**
:cl: DangerRevolution
- add: Added a Pax-injector to the Security Cyborg De-Escalation module.
- add: Added an Escalation Module for the Security Cyborg; locked behind Research.

### CODE REVIEW

**Comment by DangerRevolution** [comment]
This PR will implement the Fabricate; the refactor comes in a separate PR

File: Resources/Prototypes/DeltaV/Entities/Objects/Specific/Robotics/borg_modules.yml [hunk 1]
```diff
@@ -65,6 +65,18 @@
...
+
+- type: entity
+  id: BorgModuleSecurityEscalate
+  parent: BorgModuleSecurityDeescalate
+  name: escalation cyborg module
+  description: peace was never an option.
+  components:
+  - type: ItemBorgModule
+    items:
+    - Truncheon
```
Comment (DangerRevolution) [inline Resources/Prototypes/DeltaV/Entities/Objects/Specific/Robotics/borg_modules.yml]:
```suggestion
    - Truncheon # Anti-Fauna option to stop them being relied on the Seclite (which will be removed when the Laws menu is fixed)
```

**Comment by DangerRevolution** [comment]
code & EE dependency removed

File: Resources/Prototypes/DeltaV/Entities/Objects/Specific/Robotics/borg_modules.yml [hunk 2]
```diff
@@ -65,6 +65,22 @@
     items:
     - StunbatonSelfCharging
     - FlashRecharging
+    - BorgHypoPax
+
+- type: entity
+  id: BorgModuleSecurityEscalate
+  parent: BorgModuleSecurityDeescalate
+  name: escalation cyborg module
+  description: peace was never an option.
```
Comment (Colin-Tel) [inline Resources/Prototypes/DeltaV/Entities/Objects/Specific/Robotics/borg_modules.yml]:
```suggestion
  description: Peace was never an option.
```
:trollface:

Comment (DangerRevolution) [reply Resources/Prototypes/DeltaV/Entities/Objects/Specific/Robotics/borg_modules.yml]: Chat, what the sigma?

**Review by Colin-Tel — APPROVED** [review]
Looks good following discussion about balance. 100% can revisit this if it needs adjustments

**Pull request merged**